src/main/java/com/mcitmocks/mcitmocks/User/UserRepository.java

```diff
@@ -3,10 +3,12 @@
 import org.springframework.data.repository.CrudRepository;
 
 import java.util.Optional;
+import java.util.UUID;
```

Comment:
[nit]: This import is probably not used? I don't think this file needs any changes.